E-Commerce/readme.md

```diff
@@ -46,13 +46,27 @@ erDiagram
 Write SQL queries to complete the following tasks:
 
 - [ ] List all the products whose name contains the word "socks"
+           =>SELECT * FROM products WHERE product_name LIKE '%socks%';
+
 - [ ] List all the products which cost more than 100 showing product id, name, unit price, and supplier id
+          =>SELECT p.id AS product_id, p.product_name AS name, pa.unit_price, pa.supp_id AS supplier_id FROM products AS p INNER JOIN product_availability AS pa ON p.id = pa.prod_id WHERE pa.unit_price>100;
 - [ ] List the 5 most expensive products
+        =>select product_name, unit_price from products join product_availability on products.id = product_availability.prod_id order by unit_price desc limit 5;
 - [ ] List all the products sold by suppliers based in the United Kingdom. The result should only contain the columns product_name and supplier_name
+      => select product_name, supplier_name from products join product_availability on products.id = product_availability.prod_id join suppliers ON supp_id = suppliers.id where country = 'United Kingdom';  
```

Comment:
this is great! You have the product_name, and supplier_name-- nice job with using the joins and where clause